monitoring/api/v3/alerts-client/snippets.py

```diff
@@ -136,9 +138,11 @@ def default(self, obj):
 # [START monitoring_alert_create_channel]
```

Comment:
/FYI there probably should not be multiple codeblocks here with the same tag, if you remove them, check that there aren't extra end tags around either.

Reply:
The multiple tags have been there for a while, and the matching docs seem to be picking them up okay, so I don't want to change them now.